tools/android_test.sh

```diff
@@ -42,5 +43,15 @@ fi
 # The steps are as follows.
 
 # $ meson build [flags...]
-# $ cd build
-# $ adb push res/ /data/local/tmp/nntr_android_test
+# meson build will unzip golden data for the unit tests
+cd ../../../
+meson build -Dopenblas-num-threads=1  \
```

Comment:
there might be cases where the `build` directory already exists. we should handle that case!

Reply:
Great point! I will update the script. Thanks!